### ISSUE TITLE
ListView+Image first time  run on ios get stuck

### ISSUE THREAD

**Issue by xy221150** [flutter/flutter#69636]
flutter doctor -v
![image](https://user-images.githubusercontent.com/34728569/97946659-0d5f1f80-1dc6-11eb-9e27-c695c2d892ea.png)

video linke
[RPReplay_Final1604373139.zip](https://github.com/flutter/flutter/files/5479154/RPReplay_Final1604373139.zip)

this is code:
![image](https://user-images.githubusercontent.com/34728569/97946854-88c0d100-1dc6-11eb-9e78-757052baac62.png)
![image](https://user-images.githubusercontent.com/34728569/97946862-8f4f4880-1dc6-11eb-898d-5dd46c02480b.png)

this issue only ios,android is normal

**Comment by pedromassangocode** [comment]
Hi @xy221150 
Looks like you are using an invalid Flutter channel.
Can you please upgrade to the latest  `stable` and try again
`flutter channel stable`
`flutter upgrade --force`
`flutter doctor -v`

If the issue persists, please provide your `flutter doctor -v`, a minimal complete reproducible code sample.
Thank you

**Comment by xy221150** [comment]
I tried to update the stable but it didn't work
![image](https://user-images.githubusercontent.com/34728569/98061611-372c4b00-1e87-11eb-872d-a4d5ae812276.png)
run ios device still stuck,Whether debug or Proflie

**Comment by xy221150** [comment]
https://github.com/xy221150/img_list_demo.git
this is demo link

**Comment by pedromassangocode** [comment]
Hi @xy221150 
Please provide a code sample that does not contains third-party packages.
Thak you

**Comment by xy221150** [comment]
The code to  remove third-party packages has been uploaded, please again to pull,Thank you

**Comment by pedromassangocode** [comment]
Hi @xy221150 
I'm getting lots of exceptions running your code. But cannot reproduce the issue


```
════════ Exception caught by image resource service ════════════════════════════════════════════════
Bad state: Insecure HTTP is not allowed by platform: http://gank.io/images/f4f6d68bf30147e1bdd4ddbc6ad7c2a2
════════════════════════════════════════════════════════════════════════════════════════════════════

════════ Exception caught by image resource service ════════════════════════════════════════════════
Bad state: Insecure HTTP is not allowed by platform: http://gank.io/images/dc75cbde1d98448183e2f9514b4d1320
════════════════════════════════════════════════════════════════════════════════════════════════════

```

**Comment by xy221150** [comment]
you can try in ios/Runner/info.plist add below dictionary definition
`<key>NSAppTransportSecurity</key>
<dict>
        <key>NSAllowsArbitraryLoads</key>
        <true/>
</dict>`

**Comment by xy221150** [comment]
[IMG_2339.zip](https://github.com/flutter/flutter/files/5487548/IMG_2339.zip)
This is the demo video,which performs very badly

**Comment by pedromassangocode** [comment]
Hi @xy221150 
I'm still not able to reproduce the issue.

![gif](https://user-images.githubusercontent.com/67424416/98115237-cbf27100-1ea6-11eb-8b4a-bb2057ea76ad.gif)

Are you able to reproduce the issue on Simulator as well?
Can you provide more information related to your Physical device?

**Comment by xy221150** [comment]
![IMG_AA13A86F6E2D-1](https://user-images.githubusercontent.com/34728569/98116896-eac32880-1ee3-11eb-8d62-883b2bd8abb9.jpeg)
This my device info,I'm running is profile mode

**Comment by xy221150** [comment]
I can't replicate this problem in the simulator

**Comment by xy221150** [comment]
This problem occurs whenever an app is installed using a new emulator
![Jietu20201104-213456](https://user-images.githubusercontent.com/34728569/98118393-1f37e400-1ee6-11eb-80c8-7e92798dde09.gif)

**Comment by xy221150** [comment]
#[https://github.com/flutter/flutter/issues/65909](url)
 #[https://github.com/flutter/flutter/issues/68552](url)
 #[https://github.com/flutter/flutter/issues/45888](url)
#[https://github.com/flutter/flutter/issues/62570](url)
These issues may be very similar to what I encountered

**Comment by pedromassangocode** [comment]
Hi @xy221150 
Have you faced this on Android devices/emulator as well?

Cc @Piinks @liyuqian

**Comment by xy221150** [comment]
Android devices are normal

**Comment by chinmaygarde** [comment]
@pedromassangocode: Were you able to reproduce this?

**Comment by pedromassangocode** [comment]
@chinmaygarde no. I tried with most of my simulators and was not able to reproduce it. I don't have a physical iOS device to reproduce it.

**Comment by liyuqian** [comment]
I also can't reproduce it with a physical iPhone 6s. See the attached screenshot.
[repro69636.mov.zip](https://github.com/flutter/flutter/files/5520855/repro69636.mov.zip)

**Comment by pedromassangocode** [comment]
@xy221150 Can you please provide your physical device's information? So that anyone with the exact same device could try to reproduce the issue.

The one you provided in https://github.com/flutter/flutter/issues/69636#issuecomment-721729687 seems to be from the Simulator.

**Comment by xy221150** [comment]
![image](https://user-images.githubusercontent.com/34728569/98909006-31281100-24fc-11eb-83d0-84673ea9520c.png)

**Comment by pedromassangocode** [comment]
Hi @xy221150 
This screenshot does not provide full details. 
Can you please share the device name and model as well?

**Comment by xy221150** [comment]
![Uploading IMG_2487.PNG…]()

**Comment by xy221150** [comment]
![IMG_2487](https://user-images.githubusercontent.com/34728569/98916256-e6ab9200-2505-11eb-96be-706e0fc6a27f.PNG)
